frontend/src/components/payment-status/StatusIndicator.tsx

```
@@ -0,0 +1,133 @@
+  className = '',
+}: StatusIndicatorProps) {
+  const isPaid = status === 'paid';
+
+  // Format timestamp for display (FR-017)
+  const formattedTimestamp = timestamp && showTimestamp
+    ? formatTimestampForDisplay(timestamp)
+    : null;
+
+  return (
+    <div
+      role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)